### PR TITLE
chore: plane space sign in workflow improvement

### PR DESCRIPTION
**Pull request by anmolsinghbhatia**
This PR includes enhancements for the plane space sign in workflow.

### Sign in
image
---

### Sign in with code
image

---

### Sign in with password
image

---

### Set password link requested
image

### CODE REVIEW

File: space/components/accounts/sign-in-forms/set-password-link.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
+import React, { useState } from "react";
```
Comment (aaryan610) [inline space/components/accounts/sign-in-forms/set-password-link.tsx:1]: Update this component's logic as per the web app.

File: space/components/accounts/sign-in-forms/email-form.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+          title: "Error!",
+          message: err?.error ?? "Something went wrong. Please try again.",
+        })
+      )
+      .finally(() => setIsCheckingEmail(null));
+  };
+
+  return (
+    <>
+      <h1 className="text-center text-2xl font-semibold text-onboarding-text-100">Get on your flight deck!</h1>
```
Comment (aaryan610) [inline space/components/accounts/sign-in-forms/email-form.tsx:84]: Update the content here.

File: space/components/accounts/sign-in-forms/set-password-link.tsx (hunk 1 above)
Comment (anmolsinghbhatia) [reply space/components/accounts/sign-in-forms/set-password-link.tsx:1]: Could you provide more details about the changes you're suggesting for this component's logic? I want to ensure I understand your concerns correctly to address them accurately. Your clarification would be really helpful!

File: space/components/accounts/sign-in-forms/email-form.tsx (hunk 2 above)
Comment (anmolsinghbhatia) [reply space/components/accounts/sign-in-forms/email-form.tsx:84]: I've implemented the content provided. If there are any further updates or changes you'd like to include, please let me know. I'll make sure to review and implement any additional changes as needed.

File: space/components/accounts/sign-in-forms/set-password-link.tsx (hunk 1 above)
Comment (aaryan610) [reply space/components/accounts/sign-in-forms/set-password-link.tsx:1]: Please check the component of the web app.

**Pull request closed without merging**